Documentation/network/servicemesh/envoy-circuit-breaker.rst

```diff
@@ -0,0 +1,251 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_envoy_circuit_breaker:
+
+*******************
+L7 Circuit Breaking
+*******************
+
+Cilium Service Mesh defines a ``CiliumClusterwideEnvoyConfig`` CRD which allows users
+to set the configuration of the Envoy component built into Cilium agents.
```

Comment:
nit: agents should probably capitalized?

Reply:
I see that many other documents are `Cilium agents`, and I think it can be consistent here. I think that it is ok here.

Reply:
done.